cli.js

```diff
@@ -1103,20 +1103,34 @@ cli.progress = function (progress, decimals, stream) {
     }
     last_progress_call = now;
 
-
+    var pwr = Math.pow(10, decimals);
+    var percentageAsNum = Math.floor(progress * 100 * pwr) / pwr; 
```

Comment:
Ah no problem :smile: It's a bit inconsistent, I know. It looks like 131d0874805388d4e9d09c110d0dcddad54e3fef introduced some of the camelCase vars you're talking about, and I know switching from JS to Python has made me slip up every now and then.